.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     "max-classes-per-file": ["error", 10],
     "import/prefer-default-export": "off",
     "object-curly-newline": "off",
+    "no-restricted-syntax": ["error", "ForInStatement", "LabeledStatement", "WithStatement"],
```

Comment:
Replacing airbnb rule with following, to re-enable "ForOfStatement"

Reply:
The not allowing for loops was the most annoying